src/common/transformations/src/transformations/common_optimizations/moc_transformations.cpp

```diff
@@ -125,6 +125,7 @@ bool ov::pass::MOCTransformations::run_on_model(const std::shared_ptr<ov::Model>
         f->validate_nodes_and_infer_types();
     }
 
+    const int64_t cf_threshold = 1024;
```

Comment:
can you give explanation what is threshold? How is this value selected?
Also, I think it should be a macro, sort of configuration for ConstantFolding

Reply:
1024 is just a preliminary estimation:  usually ConstantFolding folds 3-4 operations (~250 bytes each) from xml file during one step, so to make the sum of IR + bin sizes more or less stable, we allow ConstantFolding to increase the bin size to this value.

ok, I will add the macro